Exception in threading.excepthook:
Exception ignored in thread started by: <bound method Thread._bootstrap of <Thread(Thread-50 (_run_job), stopped 11754614784)>>
Traceback (most recent call last):
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/threading.py", line 1002, in _bootstrap
    self._bootstrap_inner()
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/threading.py", line 1047, in _bootstrap_inner
    self._invoke_excepthook(self)
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/threading.py", line 1359, in invoke_excepthook
    local_print("Exception in threading.excepthook:",
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/ipykernel/iostream.py", line 559, in flush
    self.pub_thread.schedule(self._flush)
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/ipykernel/iostream.py", line 251, in schedule
    self._event_pipe.send(b"")
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/zmq/sugar/socket.py", line 696, in send
    return super().send(data, flags=flags, copy=copy, track=track)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "zmq/backend/cython/socket.pyx", line 742, in zmq.backend.cython.socket.Socket.send
  File "zmq/backend/cython/socket.pyx", line 783, in zmq.backend.cython.socket.Socket.send
  File "zmq/backend/cython/socket.pyx", line 138, in zmq.backend.cython.socket._check_closed
zmq.error.ZMQError: Socket operation on non-socket
Exception ignored in sys.unraisablehook: <built-in function unraisablehook>
Traceback (most recent call last):
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/ipykernel/iostream.py", line 559, in flush
    self.pub_thread.schedule(self._flush)
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/ipykernel/iostream.py", line 251, in schedule
    self._event_pipe.send(b"")
  File "/Users/adamimos/anaconda3/envs/epsilon-machine/lib/python3.11/site-packages/zmq/sugar/socket.py", line 696, in send
    return super().send(data, flags=flags, copy=copy, track=track)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "zmq/backend/cython/socket.pyx", line 742, in zmq.backend.cython.socket.Socket.send
  File "zmq/backend/cython/socket.pyx", line 783, in zmq.backend.cython.socket.Socket.send
  File "zmq/backend/cython/socket.pyx", line 138, in zmq.backend.cython.socket._check_closed
zmq.error.ZMQError: Socket operation on non-socket
| Epoch | Training Acc. | Loss | Overall Acc. | Last Bit Acc. |
|   1   |    49.56%    |    0.6932     |      50.51%       |      50.44%      |
|   2   |    49.85%    |    0.6930     |      49.51%       |      49.56%      |
|   3   |    50.13%    |    0.6942     |      50.43%       |      50.29%      |
|   4   |    50.00%    |    0.6933     |      49.54%       |      49.63%      |
|   5   |    50.17%    |    0.6946     |      50.37%       |      50.22%      |
|   6   |    49.77%    |    0.6931     |      49.59%       |      49.56%      |
|   7   |    49.80%    |    0.6934     |      49.53%       |      49.42%      |
|   8   |    50.24%    |    0.6933     |      49.58%       |      49.63%      |
|   9   |    50.14%    |    0.6938     |      49.58%       |      49.42%      |
[34m[1mwandb[39m[22m: Ctrl + C detected. Stopping sweep.
|  10   |    49.93%    |    0.6932     |      49.53%       |      49.56%      |